Lib/fontTools/merge.py

```diff
@@ -923,7 +976,6 @@ def _postMerge(self, font):
 @timer("make one with everything (TOTAL TIME)")
 def main(args=None):
 	from fontTools import configLogger
-
```

Comment:
I was thinking to make code more compact, but it seems these empty lines separate these the logic blocks. I'm adding them back.